homeassistant/components/overkiz/update.py

```diff
@@ -0,0 +1,113 @@
+"""Support for Overkiz updates."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyoverkiz.enums import UIClass, UIWidget, UpdateBoxStatus
+from pyoverkiz.models import Gateway
+
+from homeassistant.components.update import (
+    UpdateDeviceClass,
+    UpdateEntity,
+    UpdateEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN
+from .entity import OverkizEntity
+
+UPDATE_COMMAND = "update"
```

Comment:
Move to PyOverkiz constants eventually

Reply:
https://github.com/iMicknl/python-overkiz-api/pull/701